build/post_build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -eo pipefail
+echo "==========STARTING POST_BUILD=========="
+echo "Sleeping 180 seconds to ensure the AMI Name is searchable..."; sleep 180
```

Comment:
This sleep seems to lack context when in the post_build script. Might make more sense to keep it in the CI config file.